cnd/src/swap_protocols/halight.rs

```diff
@@ -243,7 +263,23 @@ where
     })
 }
 
-#[derive(Clone, Copy, Debug)]
+#[derive(Copy, Clone, Debug)]
 pub struct Params {
```

Comment:
Looking at the usage of `halight::Params` I think it would make sense to add the secret hash to it as they are always used together.

```suggestion
pub struct Params {
	pub secret_hash: SecretHash,
```

Reply:
@rishflab not keen on this one?

Reply:
I agree with Franck here, for other connectors the secret hash is part of the params :)